WordPress/Classes/ViewRelated/Site Creation/Services/ShoppingCartService.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+protocol ShoppingCartServiceProtocol {
+    func createSiteCreationShoppingCart(
```

Comment:
nit: the repeated word "creation" makes this less readable. Maybe `makeSiteCreationShoppingCart` would be more readable.